### ISSUE TITLE
bug: new AUnit version causes PlatformIO build failure.

### ISSUE THREAD

**Issue by sleepdefic1t** [ArkEcosystem/cpp-crypto#100]
**Expected Behavior**

All platforms build successfully.

**Current Behavior**

AUnit released v.1.3 which is causing one of their desktop-only directories to be included in the search path leading to PlatformIO builds failing.

```bash
.piolibdeps/AUnit_ID2778/unitduino/Arduino.cpp: In function 'long unsigned int millis()':
.piolibdeps/AUnit_ID2778/unitduino/Arduino.cpp:15:17: error: 'CLOCK_MONOTONIC' was not declared in this scope
clock_gettime(CLOCK_MONOTONIC, &spec);
^
.piolibdeps/AUnit_ID2778/unitduino/Arduino.cpp:15:39: error: 'clock_gettime' was not declared in this scope
clock_gettime(CLOCK_MONOTONIC, &spec);
^
*** [.pioenvs/esp32/src/test/.piolibdeps/AUnit_ID2778/unitduino/Arduino.cpp.o] Error 1
```

**Possible Solution**

I opened an issue at [AUnit \#50](https://github.com/bxparks/AUnit/issues/50), and it’s currently being investigated.

In the mean time, forcing AUnit v.1.2.1 in `test/platformio.ini` allows builds to compile successfully and tests to subsequently run and pass.

```bash
; PlatformIO Project Configuration File
;
;   Build options: build flags, source filter
;   Upload options: custom upload port, speed and extra flags
;   Library options: dependencies, extra library storages
;   Advanced options: extra scripting
;
; Please visit documentation for the other options and examples
; http://docs.platformio.org/page/projectconf.html

[env:esp32]
platform = espressif32
board = esp32dev
framework = arduino
lib_deps = AUnit@1.2.1
upload_speed = 921600
```

**Steps to Reproduce (for bugs)**

Using the current releases:
1. `cd cpp-crypto/test`
2. `pio run -e esp32 -t upload`

**Context**

I’m going to see what comes of the discussion in AUnits repo.

If it gets resolved on their end and Cpp-Crypto builds with PlatformIo, I will just close this issue at that time.

If it cannot be resolved on their end soon-ish, I will submit a PR with a fix on our end and close this issue.

**Your Environment**

- Version used: ANY
- Operating System and version: macOS
- Environment name and version (e.g. node.js v10.15.3): PlatformIO v.3.6.7
- Board: ESP32

**Comment by sleepdefic1t** [comment]
AUnit v.1.3 revealed that Cpp-Crypto's PlatformIO configuration is incorrect.

This is addressed in #101

**Comment by sleepdefic1t** [comment]
Resolved in #101
